tests/compose/test-misc-tweaks.sh

```diff
@@ -40,6 +40,7 @@ documentation: true
 EOF
 cat > config/other.yaml <<'EOF'
 recommends: true
+selinux-label-version: 1
```

Comment:
This is begging for adding a check that the labeling of a file in `/usr/etc` is correct. :)

Reply:
Done!